amgut/db/patches/0040.sql

```diff
@@ -0,0 +1,220 @@
+-- Sep. 9, 2016
+-- Plate Mapper test 01
+
+SET search_path TO ag;
```

Comment:
Remove this, I don't think it is needed. You just need to prepend the tables with the schema name (i.e. `ag.TABLE_NAME`.

Also, I think it will be useful to create a "platemapper" schema in which all the tables that you're creating are added in there. It will help to keep everything organized. SQL support FK across multiple schemas 😄 

What do you think @wasade?